#!/usr/bin/env python
PACKAGE = "image_difference"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("take_picture_table",     bool_t,   0, "It is true, the script takes a picture of the table",  False)
gen.add("successful_placement",   bool_t,   0, "If the garnment is successful placement",  False)
gen.add("save_current_img",       bool_t,   0, "It is true, the script saves current image",  False)
gen.add("threshold",              int_t,    0, "Threshold", 17, 0, 30)
gen.add("kernel_x",               int_t,    0, "Kernel X", 10, 1, 12)
gen.add("kernel_y",               int_t,    0, "Kernel Y", 10, 1, 12)
gen.add("dilate_iter",            int_t,    0, "Dilate iterations", 6, 0, 20)
gen.add("erode_iter",             int_t,    0, "Erode iterations", 8, 0, 20)


exit(gen.generate(PACKAGE, "image_difference", "Params"))
